src/lib/formats/ap2_dsk.cpp

```diff
@@ -471,19 +475,17 @@ bool a2_16sect_format::save(util::random_read_write &io, const std::vector<uint3
 				uint8_t tr = gcr4_decode(h[2],h[3]);
 				uint8_t se = gcr4_decode(h[4],h[5]);
 				uint8_t chk = gcr4_decode(h[6],h[7]);
-				#ifdef VERBOSE_SAVE
-				uint32_t post = get_u24be(&h[8]);
-				printf("Address Mark:\tVolume %d, Track %d, Sector %2d, Checksum %02X: %s, Postamble %03X: %s\n", vl, tr, se, chk, (chk ^ vl ^ tr ^ se)==0?"OK":"BAD", post, (post&0xFFFF00)==0xDEAA00?"OK":"BAD");
-				#endif
+				if constexpr (VERBOSE_SAVE) {
+					uint32_t post = get_u24be(&h[8]);
+					printf("Address Mark:\tVolume %d, Track %d, Sector %2d, Checksum %02X: %s, Postamble %03X: %s\n",
+						vl, tr, se, chk, (chk ^ vl ^ tr ^ se)==0?"OK":"BAD", post, (post&0xFFFF00)==0xDEAA00?"OK":"BAD");
```

Comment:
We use an additional indentation level for statement continuations vs scope levels, e.g.:
```c++
    if (x)
    {
        f( // indented one level because scope depth increased
                a, // two levels for continuation
                b);
        g();
    }
```

Reply:
Okay, updated.